.gitignore

```diff
@@ -12,3 +12,4 @@ website/builder/docs
 **/yarn.lock
 **/yarn-error.log
 !/yarn.lock
+.vscode/launch.json
```

Comment:
it's better add environment specific ignore into your local gitignore rather than in projects repos.